src/datasets/packaged_modules/spark/spark.py

```diff
@@ -29,6 +31,47 @@ class SparkConfig(datasets.BuilderConfig):
     features: Optional[datasets.Features] = None
 
 
+def _generate_iterable_examples(
+    df: "pyspark.sql.DataFrame",
+    partition_order: List[int] = None,
```

Comment:
`partition_order` doesn't seem to be used here ?

Reply:
Fixed